src/jsTyping/jsTyping.ts

```diff
@@ -211,37 +214,37 @@ export function discoverTypings(
         );
         addInferredTypings(module, "Inferred typings from unresolved imports");
     }
+    // Remove typings that the user has added to the exclude list

```

Comment:
This is order change and shouldnt have any impact as such on functionality a359540